src/SLCore.Listeners/Implementation/LoggerListener.cs

```diff
@@ -34,25 +34,26 @@ public class LoggerListener : ILoggerListener
     [ImportingConstructor]
     public LoggerListener(ILogger logger)
     {
-        this.logger = logger;
+        this.logger = logger.ForContext(SLCoreStrings.SLCoreName, SLCoreStrings.SLCoreInternalLogContext);
```

Comment:
I think it's enough to add just one context that is a string of type "SlCore Logs". It would occupies less space than [SLCore > Logs].